compiler/runtime/Helpers.inc

```diff
@@ -50,7 +50,8 @@ SETVAL(TR_writeBarrierBatchStore,TR_writeBarrierStoreRealTimeGC+1)
 SETVAL(TR_arrayStoreException,TR_writeBarrierBatchStore+1)
 SETVAL(TR_throwClassCastException,TR_arrayStoreException+1)
 SETVAL(TR_throwCurrentException,TR_throwClassCastException+1)
-SETVAL(TR_IncompatibleClassChangeError,TR_throwCurrentException+1)
+SETVAL(TR_throwUnreportedException,TR_throwCurrentException + 1)
+SETVAL(TR_IncompatibleClassChangeError,TR_throwCurrentException+2)
```

Comment:
Should this be `TR_throwUnreportedException+1` ?

Reply:
`TR_throwCurrentException+2` and `TR_throwUnreportedException+1` are actually equal.

Reply:
Yes, but it seems an unnecessary change in the pattern.

Reply:
changed to `TR_throwUnreportedException+1` as suggested.